apps/web/.env

```diff
@@ -6,3 +6,4 @@ GLOBAL_CONTEXT_PATH=
 WEB_CONTEXT_PATH=
 REACT_APP_INTERCOM_APP_ID=
 REACT_APP_SEGMENT_KEY=
+LOGROCKET_ID='eh8t7r/novu'
```

Comment:
shouldn't we set it from the pipeline?

Reply:
Yes, this should be removed from the .env file. And added to the deploy dev web and deploy prod web workflows on github actions.